packages/ui/src/app/pages/Proposals/ProposalPreview.tsx

```diff
@@ -84,7 +86,7 @@ export const ProposalPreview = () => {
   const myVote = proposal?.votes.find((vote) => vote.voter.id === active?.id && vote.votingRound === currentVotingRound)
   const myVoteStatus = myVote?.voteKind
 
-  if (isLoading || !proposal || !votes) {
+  if (!proposal || !votes) {
     return (
```

Comment:
Here it would be good to also have a:
```tsx
if (!isLoading) history.replace('/404')
```
before the return (just like in `@/app/pages/Bounty/Bounty`)